### PR TITLE
Fix comment workflow

### PR DESCRIPTION
**Pull request by arturo-seijas**
Fix reference to comment action

### CODE REVIEW

File: .github/workflows/comment.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     if: github.event.workflow_run.event == 'pull_request'
     steps:
       - uses: actions/checkout@v3
-      - uses: canonical/operator-workflows/.github/actions/comment_pr
+      - uses: canonical/operator-workflows/.github/actions/comment_pr@main
```
Comment (weiiwang01) [inline .github/workflows/comment.yaml:13]: I think you can use relative path here, like `./.github/actions/comment_pr`
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-an-action-in-the-same-repository-as-the-workflow

Comment (arturo-seijas) [reply .github/workflows/comment.yaml:13]: I'm checking out the indico repository and the file doesn't exist in there. In the next PR I'll move this action to a new repository,  so I'd leave it as it is for now if that works for you

**Comment by gregory-schiano** [comment]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by mthaddon — APPROVED** [review]
LGTM

**Pull request merged**